tools/clang/test/HLSLFileCheck/hlsl/operators/binary/shift-mask.hlsl

```diff
@@ -0,0 +1,33 @@
+// RUN: %dxc -T lib_6_3 %s -fcgl | FileCheck %s
+
+int shl32(int V, int S) {
+  return V << S;
+}
+
+// CHECK: define internal i32 @"\01?shl32
+// CHECK-DAG:  %[[Masked:[0-9]+]] = and i32 %{{[0-9]+}}, 31
```

Comment:
I expect you've figured this out, but for debug builds, you'll need to allow for alphabetic characters here too.